packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -212,4 +219,46 @@ contract L1Block is ISemver {
             sstore(operatorFeeConstant.slot, shr(160, calldataload(164)))
```

Comment:
@tynes looks like this will interfere with the operator fee fix, right?

Reply:
it doesn't, we don't change the type after all, but just update the computation.